multimodal/tests/unittests/others_2/test_predictor_advanced.py

```diff
@@ -20,8 +20,15 @@
         ("t5-small", LORA_NORM, "mean", "bf16-mixed", 0.00557, True),
         ("google/flan-t5-small", IA3_LORA, "mean", "bf16-mixed", 0.006865, True),
         ("google/flan-t5-small", IA3, "cls", "bf16-mixed", 0.0004201, False),
-        ("microsoft/deberta-v3-small", LORA_BIAS, "mean", "16-mixed", 0.001422, True),
-        ("microsoft/deberta-v3-small", IA3_BIAS, "cls", "16-mixed", 0.00044566, False),
+        ("microsoft/deberta-v3-small", LORA_BIAS, "mean", "32-true", 0.001422, True),
+        (
+            "microsoft/deberta-v3-small",
+            IA3_BIAS,
+            "cls",
+            "32-true",
+            0.00044566,
+            False,
+        ),  # Note: DeBERTa models need to use 32-true mixed precision to avoid overflow issues in the attention mask computation, particularly when using gradient checkpointing
```

Comment:
Can you provide a pointer regarding this issue? deberta and 16-mixed are our default best_quality model and precision, which have been used without problems.

Reply:
I guess the issue might be related to IA3, not to the deberta model.

Reply:
ok fine